### ISSUE TITLE
open() method starts default firefox browser if failed run driver provided by setWebDriver()

### ISSUE THREAD

**Issue by RocketRaccoon** [codeborne/selenide#170]
Hi,

I create `Webdriver` object and pass it to `setWebDriverRunner()` to manage drivers as described in wiki.

In some case when no browsers in grid which meet capabilities available, or local browser (e.g. chromedriver) fails to start, `open()` starts Firefox and trys to run test.

Is it feature and I could disable it, or it is a bug?

``` java
private static void  setWebDriverRunner(WebDriver driver) {
    WebDriverRunner.setWebDriver(driver);
    driver.manage().window().maximize();
}
```

**Comment by asolntsev** [comment]
I will investigate it next week, after coming back from Codefest conference.

**Comment by asolntsev** [comment]
I guess it's a feature. If there is no suitable browser yet, Selenide starts new browser. 
Sure, it can be fixed. Can you specify what should be correct behaviour? Selenide should just fail immediately?

**Comment by RocketRaccoon** [comment]
Yes, I'm expected test failure if browser specified by `WebDriverRunner.setWebDriver()`. So it looks senseable in case when user don't use default `open()` and managing WebDriver manually to have full control over process, e.g. I'm expected to run phantomjs on linux instead get Firefox on Windows.

**Comment by RocketRaccoon** [comment]
Any updates on this issue?

**Comment by asolntsev** [comment]
No yet. Probably in next Selenide version.

**Comment by rosolko** [comment]
If still affect feel free to reopen.

**Comment by asolntsev** [comment]
@RocketRaccoon @rosolko Actually it's still an issue. But it's not easy to fix.

I hope it will be fixed when https://github.com/codeborne/selenide/issues/354 gets implemented.
